Sources/SwiftSvg/Svg+Text.swift

```diff
@@ -0,0 +1,23 @@
+//
+//  Svg+Text.swift
+//  
+//
+//  Created by Aaron Ge on 2022/4/8.
+//
+
+extension Svg{
```

Comment:
Could you move out the text from the Svg scope? The SwiftSvg library already provides the module namespace so we don't need to place Svg related objects under the Svg extension. Also please provide some basic docs & unit tests if possible. Thank you. 👍

Reply:
move Text Tag out from Svg extension